src/lib/atoms/Button.svelte

```diff
@@ -50,7 +54,14 @@
   <a {...$$restProps} {href} class={disabled ? 'pointer-events-none block' : 'pointer-events-auto block'}>
     <button
       {...$$restProps}
-      class={`rounded-full px-f24 py-f8 text-center transition-all ${btnStylesClass} ${btnSizeClass} ${$$restProps.class ?? ''}`}
+      class={cn(
```

Comment:
Swap to cn here so that we can just dump in the classes and have them resolve correctly